constitution.tex

```diff
@@ -552,8 +552,18 @@
 \asubsubsubsubsection{Voting}
 On a member by member basis, this evaluation process determines if each Introductory member has successfully completed the Introductory Process requirements \ref{Expectations of an Introductory Member}.
 A Simple Majority vote with a quorum of two-thirds of the Total Number of Possible Votes is required for the evaluation to be valid.
+In order for a Active Member to be counted as a Possible Vote they must meet the requirements outlined in \ref{Expectations of Voting Members}.
 Neither absentee nor proxy votes are allowed.
 House may choose any of the Outcomes for each member.
+\asubsubsubsubsection{Expectations of Voting Members}
+The following requirements must be met by the beginning of the Introductory Evaluation for an Active Member to be allowed to vote. Any of these requirements may be waived by the Evaluations Director or a simple majority vote by the Executive Board
```

Comment:
Please comply to the style conventions https://github.com/ComputerScienceHouse/Constitution#document-style